lib/reek/ast/node.rb

```diff
@@ -34,7 +34,7 @@ def [](index)
       end
 
       def line
-        loc.line
+        loc.line unless loc.nil?
```

Comment:
Because `loc` is `nil` for cases where we send things like `s(:send, s(:self), :thing)` to `MethodContext#record_call_to`. We seem to do this only in specs (in four places), but I didn’t want to break this use case.